parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -305,6 +305,20 @@ else
         # -________________-
         systemctl restart --no-block apt-daily.service
         aptmarkWALinuxAgent unhold &
+    elif [[ $OS == $MARINER_OS_NAME ]]; then
+        {{- if EnableUnattendedUpgrade }}
+        # By default the dnf-automatic is service is notify only in Mariner.
+        # Enable the automatic install timer and the check-restart timer.
+        # Stop the notify only dnf timer since we've enabled the auto install one.
+        # systemctlDisableAndStop adds .service to the end which doesn't work on timers.
+        systemctl disable dnf-automatic-notifyonly.timer
+        systemctl stop dnf-automatic-notifyonly.timer
+        # At 6:00:00 UTC (1 hour random fuzz) download and install package updates.
+        systemctl unmask dnf-automatic-install.service || exit $ERR_SYSTEMCTL_START_FAIL
+        systemctl unmask dnf-automatic-install.timer || exit $ERR_SYSTEMCTL_START_FAIL
+        systemctlEnableAndStart dnf-automatic-install.timer || exit $ERR_SYSTEMCTL_START_FAIL
```

Comment:
asssume systemctlEnableAndStart is alias on mareiner for enable not sure why @Ace used restart instead of start.

Reply:
They are helper functions, part of https://github.com/dmcilvaney/AgentBaker/blob/82f8ea4937c2cf8ead85b53d109fa836988ae574/parts/linux/cloud-init/artifacts/cse_helpers.sh#L225

If I had to guess, running without `--no-block` was causing issues with the apt service taking a long time to start. Might add that if anything shows up in testing, but since we are just starting/stopping times I don't expect to need to run asynchronously here.